### PR TITLE
[kv:get] pretty print JSON

### PR DESCRIPTION
**Pull request by bhageena**
[kv:get] pretty print JSON value if stored value is in JSON format

### CODE REVIEW

File: src/commands/key-value/get.ts [hunk 1]
```diff
@@ -22,22 +22,28 @@ export default class Get extends NimBaseCommand {
...
-            logger.handleError(
+      await queryKVStore(queryCommand, args, flags, authPersister)
+        .then(res => {
+          try {
+            logger.logJSON(JSON.parse(res.value))
+          } catch (e) {
+            logger.log(res.value)
+          }
+        })
+      // Log the error returned by the action.
```
Comment (rabbah) [inline src/commands/key-value/get.ts:42]:
```suggestion
        // Log the error returned by the action.
```

**Review by rabbah — APPROVED** [review]
Thanks for the quick patch.

**Comment by rabbah** [comment]
Shoud/Can we enable the linter to run automatically on PRs?

**Review by joshuaauerbachwatson — APPROVED** [review]
LGTM

**Pull request merged**